src/game/boe.items.hpp

```diff
@@ -37,7 +37,22 @@ void reset_item_max();
 short item_val(cItem item);
 void place_treasure(location where,short level,short loot,short mode);
 std::string get_text_response(std::string prompt = "", pic_num_t pic = 16);
-short get_num_response(short min, short max, std::string prompt, std::vector<std::string> choice_names = {});
+// Prompt the player for a number, which might be an index in a given list of strings.
+// Specify cancel_value to show a cancel button, which will return the given value (for example, -1)
+short get_num_response(short min, short max, std::string prompt, std::vector<std::string> choice_names = {}, boost::optional<short> cancel_value = boost::none);
 
-short char_select_pc(short mode,const char *title);
-short select_pc(short mode);
+enum class eSelectPC {
+	ANY,
+	ONLY_LIVING,
+	ONLY_LIVING_WITH_ITEM_SLOT,
+	// Slightly different from the previous: this now checks not just for a slot,
+	// but also for carrying capacity, and will stack charges even if all slots are full.
+	// When you use this, you have to set the global item_store to your item, first.
+	ONLY_CAN_GIVE,
+	// Same as the previous, but in combat, only show *adjacent* living PCs who can take the item.
+	ONLY_CAN_GIVE_FROM_ACTIVE,
+	ONLY_DEAD,
+};
```

Comment:
This was originally `enum class eSelectPC { ONLY_LIVING, ANY, ONLY_DEAD };`, right? Reordering it probably  breaks the Select Target special node.

Reply:
That didn't need to be fixed, because select target nodes already translated between their own integer mapping and the call args to `select_pc`:

https://github.com/NQNStudios/cboe/blob/99eb9962b5063a8bcfbe9adf2b2d4a9874ff6c65/src/game/boe.specials.cpp#L2701

There's no casting integers to the enum type.